### ISSUE TITLE
Issues in node

### ISSUE THREAD

**Issue by xjamundx** [tracker1/safe-clone-deep#2]
When using from node 4 or 6 I get the following:

```
> require('safe-clone-deep')
Error: Cannot find module 'Buffer'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/jamuferguson/dev/node_modules/safe-clone-deep/src/index.js:3:14)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
> 
```

Does this module only work in the browser?

**Comment by tracker1** [comment]
Is this still an issue for you?

**Comment by LukeBriggs4Roads** [comment]
@tracker1 this is still an issue (it occurs on Linux/ case sensitive file systems) - fixed in the above pull request

**Comment by tracker1** [comment]
@LukeBriggs4Roads Thanks...

@xjamundx the fix has been merged and published 1.1.4
